src/main/java/frc/robot/subsystems/SuperStructure.java

```diff
@@ -22,75 +22,127 @@ public SuperStructure(
-        elevator.setLevel(ElevatorSubsystem.LEVEL_TWO_POS),
+        elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_TWO_POS),
         spinnyClaw.holdExtakePower().withTimeout(0.15),
         stow());
   }
 
   public Command levelOne(BooleanSupplier score) {
     return Commands.sequence(
         Commands.parallel(
-            elevator.setLevel(ElevatorSubsystem.LEVEL_ONE_POS),
-            armPivot.moveToPosition(ArmPivot.PRESET_L1),
+            elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_ONE_POS),
+            armPivot.moveToPosition(ArmPivot.CORAL_PRESET_L1),
             spinnyClaw.stop()),
         Commands.waitUntil(score),
-        elevator.setLevel(ElevatorSubsystem.LEVEL_TWO_POS),
+        elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_TWO_POS),
         spinnyClaw.holdExtakePower().withTimeout(0.15),
         stow());
   }
 
   public Command stow() {
     return Commands.parallel(
-        elevator.setLevel(ElevatorSubsystem.STOWED),
-        armPivot.moveToPosition(ArmPivot.PRESET_STOWED),
+        elevator.setLevel(ElevatorSubsystem.CORAL_STOWED),
+        armPivot.moveToPosition(ArmPivot.CORAL_PRESET_STOWED),
         spinnyClaw.stop());
   }
 
   public Command intake() {
     return Commands.sequence(
             Commands.parallel(
-                elevator.setLevel(ElevatorSubsystem.PRE_INTAKE),
-                armPivot.moveToPosition(ArmPivot.PRESET_DOWN),
+                elevator.setLevel(ElevatorSubsystem.CORAL_PRE_INTAKE),
+                armPivot.moveToPosition(ArmPivot.CORAL_PRESET_DOWN),
                 spinnyClaw.intakePower()),
             // Commands.waitUntil(armSensor.inTrough()),
-            elevator.setLevel(ElevatorSubsystem.INTAKE),
+            elevator.setLevel(ElevatorSubsystem.CORAL_INTAKE_POS),
             Commands.waitSeconds(0.1),
             spinnyClaw.stop(),
-            elevator.setLevel(ElevatorSubsystem.PRE_INTAKE),
+            elevator.setLevel(ElevatorSubsystem.CORAL_PRE_INTAKE),
             stow())
         .withName("Intake");
   }
+
+  public Command algaeLevelThreeFourIntake() {
+    return Commands.sequence(
+        Commands.parallel(
+            spinnyClaw.algaeIntakePower(),
+            armPivot.moveToPosition(ArmPivot.ALGAE_REMOVE),
+            elevator.setLevel(ElevatorSubsystem.ALGAE_LEVEL_THREE_FOUR)),
+        algaeStow());
+  }
+
+  public Command algaeLevelTwoThreeIntake() { // theoretically
+    return Commands.sequence(
+        Commands.parallel(
+            spinnyClaw.algaeIntakePower(),
+            armPivot.moveToPosition(ArmPivot.ALGAE_REMOVE),
+            elevator.setLevel(ElevatorSubsystem.ALGAE_LEVEL_TWO_THREE)),
+        algaeStow());
+  }
+
+  public Command algaeLevelThreeFourFling() {
+    return Commands.sequence(
+        Commands.parallel(
+            spinnyClaw.algaeFlingPower(),
+            armPivot.moveToPosition(ArmPivot.ALGAE_REMOVE),
+            elevator.setLevel(ElevatorSubsystem.ALGAE_LEVEL_THREE_FOUR)),
+        algaeStow());
```

Comment:
What does the fling process look like? I'm guessing we'll flinging out the algae, but I might be misunderstanding.
Rotating the arm would mean that this should look like the coral presets with a `Commands.parallel()` to go to the pre-score position, a `Commands.waitUntil()` to wait until the driver says to score, `armPivot.moveToPosition()` to rotate the arm and hit the algae, and then going back to stow. (The factory would have a `BooleanSupplier` parameter to say when to score, and we'd want different elevator heights (since instead of aligning to the height of the algae we align to below the algae) and separate `ALGAE_PRE_FLING` and `ALGAE_FLING` constants for the arm pivot angles.